peachjam/templates/peachjam/_taxonomies.html

```diff
@@ -1,15 +1,20 @@
 {% load peachjam i18n %}
 
-<div class="col-md">
-  <h5>{% translate 'Taxonomies' %}</h5>
-  {% for taxonomy in taxonomies %}
+<h5>{% translate 'Taxonomies' %}</h5>
+{% for taxonomy in taxonomies %}
+  <div class="d-flex pt-2">
```

Comment:
The padding between the "Taxonomies" heading and the div, and the "Relationships" heading and its div, is now different because of the `pt-2`.

The h5 already provides bottom padding which I think is sufficient, and consistent for the page.

The relationships list uses `ul.list-unstyled` (which semantically is correct -- eg for screen readers -- because it's a list) which provides consistent spacing between items. I think that would be a better choice here than custom padding.

Reply:
What the taxonomies look like now on the document detail page

![image](https://user-images.githubusercontent.com/15995210/201325262-8ac3c6dd-8a8e-4500-b895-1e1ef590f7b1.png)

Reply:
refactored